src/app/wines/page.tsx

```diff
@@ -0,0 +1,48 @@
+// 와인 목록 페이지
+import WineCard from "@/components/AboutWine/WineCard";
+
```

Comment:
WindCard 컴포넌트를 import하는 경로는 "about-wine" 이지 않나요?!

```suggestion
import WineCard from "@/components/about-wine/WineCard";
```